BUILDING.md

```diff
@@ -115,3 +115,12 @@ the registry binary generated in the "./bin" directory:
 
 Optional [build tags](http://golang.org/pkg/go/build/) can be provided using
 the environment variable `DOCKER_BUILDTAGS`.
+
+<dl>
+<dt>noresumabledigest</dt>
+<dd>Compiles without resumable digest support</dd>
+<dt>include_gcs</dt>
+<dd>Adds support for <a href="https://cloud.google.com/storage">Google Cloud Storage</a></dd>
+<dt>include_oss</dt>
+<dd>Adds support for <a href="https://www.alibabacloud.com/product/object-storage-service">Alibaba Openject Storage Service</a></dd>
```

Comment:
Was `Openject` the actual  name or was that meant to be "Alibaba Cloud Object Storage Service (OSS)" (as mentioned on that page?)

```suggestion
<dd>Adds support for <a href="https://www.alibabacloud.com/product/object-storage-service">Alibaba Cloud Object Storage Service (OSS)</a></dd>
```

Reply:
Not sure how I messed that up - bad copy paste I guess :/

Thanks!

Reply:
Thanks! I wasn't immediately sure (could've been some brand-name `Openject`, so thought I'd ask just to be sure 😂